meetings/meetings/urls.py

```diff
@@ -13,7 +13,8 @@
 urlpatterns = [
     url(r'^', include(router.urls)),
     url(r'^conferences/', include('conferences.urls', namespace='conferences')),
-    url(r'^checklogin/', apiViews.checkLoggedIn.as_view(), name='checklogin'),
+    url(r'^checklogin/', apiViews.checkLoggedIn.as_view(), name='checklogin'), 
+    url(r'^current/', apiViews.viewCurrentUser.as_view(), name='current'),
```

Comment:
Both the original and new names are unpythonic, but really it probably should be `check-login` to conform to JSON API view ideas. Could we combine `check-login` with `current` into something like `current-user` that would both check if they are logged in and return information?